doc/news/changes/minor/20190814ReneGassmoeller

```diff
@@ -0,0 +1,4 @@
+Added: Add a Particles::Generator namespace and a first implementation that
```

Comment:
```suggestion
Added: Add a Particles::Generators namespace and a first implementation that
```
But we can also go with a follow-up pull request to avoid rerunning the tester.